src/Queue/JobAttempts.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Laravel\Vapor\Queue;
+
+use Illuminate\Contracts\Cache\Repository as Cache;
+
+class JobAttempts
+{
+    /**
+     * The cache repository implementation.
+     *
+     * @var \Illuminate\Contracts\Cache\Repository
+     */
+    protected $cache;
+
+    /**
+     * Create a new job attempts instance.
+     *
+     * @param  \Illuminate\Contracts\Cache\Repository  $cache
+     * @return void
+     */
+    public function __construct(Cache $cache)
+    {
+        $this->cache = $cache;
+    }
+
+    /**
+     * Increment the number of times the job has been attempted.
+     *
+     * @param  \Laravel\Vapor\Queue\VaporJob  $job
+     * @return void
+     */
+    public function increment($job)
+    {
+        $this->cache->increment($this->key($job));
```

Comment:
I think you cannot increment a key the doesn't exist yet in DynamoDB. Can you verify that?

Reply:
Fixed.